scipy/linalg/flapack_other.pyf.src

```diff
@@ -42,15 +42,54 @@ subroutine <prefix2>gejsv(joba,jobu,jobv,jobr,jobt,jobp,m,n,a,lda,sva,u,ldu,v,ld
 
 end subroutine <prefix2>gejsv
 
+subroutine <prefix2>trexc(wantq,n,a,lda,q,ldq,ifst,ilst,work,info)
+    ! Reorder the Schur decomposition of a real matrix
+    ! using an orthogonal or unitary equivalence transformation.
+
+    callstatement { ifst++; ilst++; (*f2py_func)((wantq?"V":"N"),&n,a,&lda,q,&ldq,&ifst,&ilst,work,&info); }
```

Comment:
We try to stay away from this `++` business as it often makes it more difficult instead of easy if you want to take one input or output from one routine to another. Better leave it as is and deal with it elsewhere.

Reply:
Is elsewhere in the code of the user? So if they want to swap row 1 and 2 (in 0 based indices) they have to pass 2 and 3 instead?

Reply:
Yes pretty much if you work with LAPACK routine you comply with Fortran stuff. So that strange conversions don't get in the way.